src/pages/smart-contracts/hello-world-tutorial.md

```diff
@@ -182,7 +182,7 @@ The following set of commands will achieve the same goals as the above workflow.
 Install an early release of the new Blockstack CLI for Stacks 2.0.
 
 ```bash
-sudo npm install -g "https://github.com/blockstack/cli-blockstack#feature/stacks-2.0-tx"
+npm install --global "https://github.com/blockstack/cli-blockstack#feature/stacks-2.0-tx"
```

Comment:
I was actually thinking about that and I settled on `npm` installs only because:
- we'd want to be consistent with that everywhere (even in beginners tutorials/guides)
- if we refer to yarn, we need to add that to the requirements sections in tutorials

I would rather keep it simple and only use npm and list node as req. Folks who know yarn should also be able to take the npm command and do it with yarn. wdyt?

Reply:
tl;dr - ok to move ahead without yarn for now. in the future, we might get a component to toggle so users can select what they want